form_jsons/public/value-to-the-community.json

```diff
@@ -85,10 +85,10 @@
   ],
   "version": 2,
   "skipSummary": false,
-  "name": "Apply for funding to save a building in your community",
+  "name": "Apply for funding to save an asset in your community",
```

Comment:
Not now, but for future funds we should really find a way to store this text in just one place!

Reply:
Agreed!